adoc/advanced_topics_image_management.adoc

```diff
@@ -377,7 +377,9 @@ For more information about the Kiwi build system, see the https://doc.opensuse.o
 [[at.images.kiwi.requirements]]
 === Requirements
 
-The Kiwi image building feature is available for Salt minions running {sls}{nbsp}12 or later.
+The Kiwi image building feature is available for Salt minions running {sls}{nbsp}12.
+If you want to build {sle} 15 images (when available), the Kiwi image build system must run on a {sle} 15 build host.
```

Comment:
Keep in mind that SLE15 images are not yet implemented

Reply:
ok, I think it is better if I mostly remove that statement. And the other half is commented now.

If marketing or product management wants something different, you are encouraged to provide a patch ;)